src/services/FetchPokemon.js

```diff
@@ -0,0 +1,46 @@
+// only render these 10 on load so all 801 pokemon do not flood site
```

Comment:
Great note!! For future reference, you can use pagination to limit the number of returned pokemon from the API. 😁